indico/modules/auth/templates/emails/account_creation_email.txt

```diff
@@ -0,0 +1,17 @@
+{% extends 'emails/base.txt' %}
+
+{% block subject -%}
+    {% trans %}Account has been created{% endtrans %}
+{%- endblock %}
+
+{% block header_recipient -%}
+    {{ recipient.full_name }}
+{%- endblock %}
+
+{% block body -%}
+    {% filter dedent %}
+        A new Indico profile has been created for a user with name {{ user.full_name }} ({{ user.email }})
```

Comment:
I would show the information in new lines. "for a user with name" isn't very good english.